pkg/cmd/server/kubernetes/master.go

```diff
@@ -118,6 +118,8 @@ func (c *MasterConfig) RunPersistentVolumeAttachDetachController(client *kclient
 			c.CloudProvider,
 			kctrlmgr.ProbeAttachableVolumePlugins(s.VolumeConfiguration),
 			nil,
+			s.DisableAttachDetachReconcilerSync,
```

Comment:
`ReconcilerSyncLoopPeriod` is 5 sec, `DisableAttachDetachReconcilerSync` is false. The former is tested in pkg/cmd/server/kubernetes/master_config_test.go, I'll add one another for the latter in that case as a followup.